docs/content/community/radius-artifacts.md

```diff
@@ -0,0 +1,29 @@
+---
+type: docs
+title: "Artifacts on Radius"
+linkTitle: "Artifacts on Radius"
+description: "Information about press, blog posts, presentation and other artifacts on Radius"
+weight: 400
+---
+
+## Microsoft Blog posts
+
+- [Radius Launch Oct 2023 - The Microsoft Azure Incubations Team launches Radius, a new open application platform for the cloud](https://azure.microsoft.com/en-us/blog/the-microsoft-azure-incubations-team-launches-radius-a-new-open-application-platform-for-the-cloud/) 
+- [Microsoft Open Source Blog Oct 2023 - Enabling developer collaboration with Radius](https://cloudblogs.microsoft.com/opensource/2023/10/18/enabling-developer-collaboration-with-radius/)
+- Ongoing posts at [Radius Blog](https://blog.radapp.io)
+
+## Press
+
+- [ Microsoft launches Radius, an open-source application platform for the cloud-native era - Techcrunch Oct 18,2023](https://techcrunch.com/2023/10/18/microsoft-launches-radius-an-open-source-application-platform-for-the-cloud/) 
+- [Microsoft Reimagines Modern Application Deployment With Radius Platform Forbes Oct 23,2023](https://www.forbes.com/sites/janakirammsv/2023/10/23/microsoft-reimagines-modern-application-deployment-with-radius-platform/?sh=6a78a37c76d7)
+- [Microsoft’s Radius and the future of cloud-native development - InfoWorld Oct 26,2023](https://www.infoworld.com/article/3709448/microsofts-radius-and-the-future-of-cloud-native-development.html)
+- [Microsoft Azure introduces Radius, an open-source development platform for multi-cloud computing - ZDNET Oct 27,2023](https://www.zdnet.com/article/microsoft-azure-introduces-radius-an-open-source-development-platform-for-multi-cloud-computing/)
+
+## Videos
```

Comment:
Let's move this to the second slot, as the articles page is going to be super long over time and we want the shorter videos section to stand out.